resources-domain/lambdas/search-index/index.ts

```diff
@@ -140,26 +145,39 @@ export const handler = async (event: SQSEvent): Promise<SQSBatchResponse> => {
     response.batchItemFailures.push({
       itemIdentifier: documentIdToMessageId[documentId],
     });
-
+  let fullyQualifiedResources = ['not determined'];
```

Comment:
Double-checking this is not a user-facing string that needs localization?

Reply:
No it's only for logging